### PR TITLE
Contact us info translation

### LINKED ISSUE #146: contact us leave use a message translation

### PR DESCRIPTION
**Pull request by hiba-machfej**
close #146
Screen Shot 2020-11-21 at 4 10 02 PM

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -316,6 +316,13 @@
     },
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
+    },
+    "contactUsInfo": {
+      "header": "Leave us a message",
+      "description": "Torquatos nostros? quos tu tam crudelis fuisse, nihil oportere nimium nos causae confidere, sed animo etiam erga nos causae confidere, sed uti oratione perpetua malo quam ostendis sed quia dolor repellendus temporibus autem.",
```
Comment (louisrli) [inline public/locales/en/translation.json]: is this placeholder text? i think its a little late, it detracts from your demo. can you put something real here?

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -316,6 +316,13 @@
...
+      "location": "9094 Bay Meadows Street Conyers, GA 30012",
+      "phone": "+229-955-5388-336",
```
Comment (louisrli) [inline public/locales/en/translation.json:331]: im wondering this again, if you delete this from the AR and TR translations, does it fallback to the english one? it should i believe, then you dont have to copy-paste redundant info

Comment (hiba-machfej) [reply public/locales/en/translation.json:331]: it works I think when I remove these lines from Ar, it is switching the website to Arabic. But navbar is making everything fall into English Jamil will check that issue.

File: public/locales/en/translation.json (hunk 1 above)
Comment (hiba-machfej) [reply public/locales/en/translation.json]: Done

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -323,6 +323,13 @@
     },
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
+    },
+    "contactUsInfo": {
+      "header": "Leave us a message",
+      "description": "Safe place is here for you. Please use this form to contact us, share your feedback, submit a request for assistance or work toghther to end gender-based violence.",
```
Comment (louisrli) [inline public/locales/en/translation.json]: Safe place or Safe Place? i think the latter

Comment (louisrli) [reply public/locales/en/translation.json]: typo in together

File: public/locales/en/translation.json (hunk 2 above)
Comment (louisrli) [reply public/locales/en/translation.json:331]: can you get a TR translation from someone in this PR too, or wait i forgot does this site have tr translations

File: public/locales/en/translation.json (hunk 3 above)
Comment (hiba-machfej) [reply public/locales/en/translation.json]: done

**Review by jennetturkkan — APPROVED** [review]

File: public/locales/en/translation.json (hunk 2 above)
Comment (hiba-machfej) [reply public/locales/en/translation.json:331]: no TR for us :D

Comment (hiba-machfej) [reply public/locales/en/translation.json:331]: Anadili Türkçe olan birisi yok.

**Review by louisrli — APPROVED** [review]

**Comment by hiba-machfej** [comment]
@Ammar-64

**Pull request merged**